runtime-modules/service-discovery/src/lib.rs

```diff
@@ -10,8 +10,8 @@
 //!
 //! ## Supported extrinsics
 //!
-//! - [set_ipns_id](./struct.Module.html#method.set_ipns_id) - Creates the AccountInfo to save an IPNS identity for the storage provider.
-//! - [unset_ipns_id](./struct.Module.html#method.unset_ipns_id) - Deletes the AccountInfo with the IPNS identity for the storage provider.
+//! - [set_ipns_id](./struct.Module.html#method.set_ipns_id) - Creates the StorageProviderAccountInfo to save an IPNS identity for the storage provider.
```

Comment:
I understand the need to change the name here because it conflicts with the balances::AccountInfo struct. And I recommend a better name of `ServiceProviderRecord` because it shouldn't be storage provider specific. I'm not sure how much longer we will keep this module around or if it will be used by other services, but until then we might as well use a general name.

Reply:
types library is prepared with this requested change https://github.com/Joystream/joystream/pull/1153

This is the only change request otherwise ready to merge. :+1:

Reply:
Done.